### ISSUE TITLE
no UI for normal users

### ISSUE THREAD

**Issue by guizmoau** [YunoHost-Apps/wireguard_ynh#66]
### Describe the bug

More than a bug it might be a misunderstanding or new-feature...
Users with no admin privileges can modify configuration

### Context

- Hardware: VPS
- YunoHost version: 4.3.6.3
- I have access to my server: ssh webadmin
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
- Using, or trying to install package version/branch: 0.3.7

### Steps to reproduce

- install the app
- connect with the admin user (the one selected during install)
- connect in another browser with a normal user (another one)

### Expected behavior

- the normal user has access to all admin configuration instead of just creating new client only

**Comment by tituspijean** [comment]
I'm sorry, your bug description and expected behavior are not opposite and it confuses me. Can you rephrase them?

---

From the README:

> Use YunoHost permissions panel to allow users to access the web UI.

*i.e.* if you give users access to WireGuard's UI, they can alter all its settings.
Upon installation, you choose one admin. They are by default the only one able to access the web UI.

- If you did not add the second user and they have access to it, that's an issue at YunoHost's level.
- If you added the second user and they can modify all configuration, that's expected behavior.
- If you added the second user and they can only create new clients, that's not a known feature of WireGuard's UI nor one implemented in this package. Please explain and support your claims with screenshots. :)

**Comment by guizmoau** [comment]
Ok I must have not read that part of the README carrefully enough.

I was looking for another kind of feature with Wireguard where users can freely create heir own VPN client accounts.

I might try to package [wg-access](https://github.com/Place1/wg-access-server) into yunohost...

Sorry for the shared confusion

**Comment by tituspijean** [comment]
No worries!

> I might try to package [wg-access](https://github.com/Place1/wg-access-server) into yunohost...

You are more than welcome to do so :)
